Test Suite is adding space at the top of the webpage in Personal Portfolio

<!-- freeCodeCamp Testable Projects Issue Template -->

<!-- Please provide as much detail as possible for us to fix your issue -->
<!-- Remove any heading sections you did not fill out -->

#### Issue Description
<!-- Describe below when the issue happens and how to reproduce it -->
I am not quite sure at what point the issue occurred, but for some reason the "the height of the welcome section should be equal to the height of the viewport" test will not pass for me on the Personal Portfolio project. When I hide the navbar (visibility: none), I can see that there is space above the welcome section when there should not be (I had originally done nothing to change the positioning of this section when I first noticed the failure, though I did try translating the welcome section to pass the test to no avail). I believe this issue is being caused by the test suite, as I encountered it in at least one test before and it seemed to have sorted itself out. I cannot recall what fixed the issue last time.

#### Browser Information
<!-- Describe your workspace in which you are having issues-->

* Browser Name, Version: Google Chrome, Version 56.0.2924.87 (64-bit)
* Operating System: Windows 10 Home, 64-bit
* Mobile, Desktop, or Tablet: Desktop

#### Your Code
<!-- Paste and relevant code in here -->
Error:
`AssertionError: The height of #welcome-section is not equal to the height of the viewport : expected 495 to equal 478
    at n.<anonymous> (https://gitcdn.link/cdn/freeCodeCamp/testable-projects-fcc/b9a6558063e5349870a4123cceeb72f27d6fd9b9/build/bundle.js:20104:36)
    at r (https://cdnjs.cloudflare.com/ajax/libs/mocha/3.0.2/mocha.min.js:2:7852)
    at r.run (https://cdnjs.cloudflare.com/ajax/libs/mocha/3.0.2/mocha.min.js:2:8853)
    at i.runTest (https://cdnjs.cloudflare.com/ajax/libs/mocha/3.0.2/mocha.min.js:2:13553)
    at https://cdnjs.cloudflare.com/ajax/libs/mocha/3.0.2/mocha.min.js:2:14192
    at r (https://cdnjs.cloudflare.com/ajax/libs/mocha/3.0.2/mocha.min.js:2:13024)
    at https://cdnjs.cloudflare.com/ajax/libs/mocha/3.0.2/mocha.min.js:2:13000
    at n (https://cdnjs.cloudflare.com/ajax/libs/mocha/3.0.2/mocha.min.js:2:12791)
    at https://cdnjs.cloudflare.com/ajax/libs/mocha/3.0.2/mocha.min.js:2:12855
    at i (https://cdnjs.cloudflare.com/ajax/libs/mocha/3.0.2/mocha.min.js:1:559)`

Relevant HTML:
```
 <html>
  <link href="https://fonts.googleapis.com/css?family=Rubik" rel="stylesheet">
  <body>
    <nav id="navbar">
      <h1>Jake Steele</h1>
      <a href="#welcome-section">Top</a>
      <a href="#projects">Projects</a>
      <a href="#contact">Contact</a>
    </nav>
    <main>
      <div id="welcome-section">
        <h1 class="intro-text">Hi. My name is Jake Steele.</h1>
        <h3 class="intro-text">I dabble in web design.</h3>
      </div>
    </main>
  </body>
</html>
```

Relevant CSS:
```
@media (maxheight: 640) {
  font-size: 12px;
}

*, *:before, *:after {
  box-sizing: inherit;
}

html {
  box-sizing: border-box;
  font-family: 'Rubik', sans-serif;
  background-color: #3C657F;
}

break {
  flex-basis: 100%;
  width: 0px;
  height:0px;
  overflow: hidden;
}

a {
  text-decoration: none;
}

a:hover {
  text-decoration: underline;
}

nav {
  position: fixed;
  top: 0px;
  left: 0px;
  z-index: 1;
  display: flex;
  flex-direction: row;
  justify-content: flex-end;
  align-items: center;
  height: 10vh;
  width: 100vw;
  background-color: #60A2CC;
  border-bottom: solid #3A617A 1px;
  padding: 0vh 10vw;
}

nav > a {
  color: #FFF;
  line-height: 10vh;
  font-weight: 700;
  margin: 0vh 1vw 0vh 0vw;
}

nav > h1 {
  padding-right: 72vw;
}

#welcome-section {
  top: 0px;
  left: 0px;
  height: 100vh;
  width: 100vw;
  background-color: #78CAFF;
  text-align: center;
  border-radius: 10px;
}

.intro-text {
  position: relative;
  top: 50%;
  left: 50%;
  transform: translate(-50%, -50%);
}
```


If this is just an error with my code, I'm sorry for wasting your time and I'll close the issue! Thank you!

--- a/build/bundle.js
+++ b/build/bundle.js
@@ -20114,7 +20114,7 @@ var FCC_Global =
 	            });
 
 	            it('2. The height of the welcome section should be equal to the height of the viewport.', function () {
-	                FCC_Global.assert.equal(document.getElementById('welcome-section').offsetHeight, document.documentElement.clientHeight, 'The height of #welcome-section is not equal to the height of the viewport ');
+	                FCC_Global.assert.equal(document.getElementById('welcome-section').offsetHeight, window.innerHeight, 'The height of #welcome-section is not equal to the height of the viewport ');
 	            });
 
 	            it('3. The navbar should always be at the top of the viewport.', function () {

--- a/src/project-tests/portfolio-tests.js
+++ b/src/project-tests/portfolio-tests.js
@@ -107,7 +107,7 @@ export default function createPortfolioTests() {
             });
 
             it('2. The height of the welcome section should be equal to the height of the viewport.', function() {
-                FCC_Global.assert.equal(document.getElementById('welcome-section').offsetHeight, document.documentElement.clientHeight,
+                FCC_Global.assert.equal(document.getElementById('welcome-section').offsetHeight, window.innerHeight,
                     'The height of #welcome-section is not equal to the height of the viewport ');
             });
 

